.pre-commit-hooks.yaml

```diff
@@ -12,4 +13,4 @@
   language: docker_image
   entry: zavoloklom/dclint:3.0.0
   types: [ yaml ]
-  files: ^(docker-)?compose\.ya?ml$
\ No newline at end of file
+  files: ^(docker-)?compose\.ya?ml$
```

Comment:
I tried to revert this change but I wasn't able (I tried to edit the file with VIM and IDEA, same results)